### ISSUE TITLE
Properties not displayed for components that are placed per Copy&Paste.

### ISSUE THREAD

**Issue by sabicalija** [asterics/WebACS#22]
Properties of elements that are created via `CTRL+C`, `CTRL+V` are not displayed correctly.

To reproduce this issue, perform following steps:
 1. Select a component already placed inside ACS (Model Designer).
 2. Enter `CTRL+C`, `CTRL+V`
 3. Select the newly created element.
 4. Properties are not displayed

Workaround: Don't use `CTRL+C`, `CTRL+V` for components creation/placement.

**Comment by sabicalija** [comment]
I'm having problems to copy/paste components in a model. Tested with Chrome and Firefox. Does anyone else have this problem?

**Comment by kingcole** [comment]
hmm...same here - strange, because this definitely used to work

**Comment by sabicalija** [comment]
Well, I can copy that. :relaxed: It used to work here too.

Maybe there are API changes in newer browser versions. What do you think? What might cause this problem?

**Comment by kingcole** [comment]
Yes, that might be - seems no keypress-events are fired any more for CTRL-c/v, while other keypresses actually do fire. This could be a pain in the a... - can't use keydown, for it fires as soon as you press CTRL,  can't use keyup since it fires too late...
